zlibWrapper/examples/zwrapbench.c

```diff
@@ -130,7 +130,7 @@ void BMK_SetBlockSize(size_t blockSize)
 **********************************************************/
 typedef struct
 {
-    const char* srcPtr;
+    char* srcPtr;
```

Comment:
The issue here is that `z_const` and `ZLIB_CONST` were introduced in zlib 1.2.6 and we want to support 1.2.3+.